source/agora/consensus/EnrollmentManager.d

```diff
@@ -692,6 +702,61 @@ public class EnrollmentManager
         }
     }
 
+    /***************************************************************************
+
+        Save data of integer type into the table
+
+        Params:
+            name = the name for the data
+            value = the value of the data
+
+    ***************************************************************************/
+
+    private void saveIntegerData (string name, size_t value) @trusted nothrow
+    {
+        try
+        {
+            this.db.execute("REPLACE into node_enroll_data " ~
+                "(key, val) VALUES (?, ?)", name, value);
+        }
+        catch (Exception ex)
+        {
+            log.error("ManagedDatabase operation error {}", ex);
+        }
+    }
+
+    /***************************************************************************
+
+        Load data of integer type from the table
+
+        Params:
+            name = the name for the data
+            default_value = the default value used when the data is missing
+
+        Returns:
+            the value of the data
+
+    ***************************************************************************/
+
+    private size_t loadIntegerData (string name, size_t default_value = 0)
```

Comment:
If these two methods are supposed to be generic (load/save "any" integer data), then why not also use them in `getNextRevealHeight` and `setNextRevealHeight`?

It seems inconsistent to have generic methods but they're only used in one call.

Reply:
I will use it for the two functions in another commit soon.